src/hardware/serial_terminal.rs

```diff
@@ -1,5 +1,65 @@
 use super::UsbBus;
+use crate::hardware::flash::FlashSettings;
+use crate::hardware::flash::Settings;
 use core::fmt::Write;
+use miniconf::{JsonCoreSlash, TreeKey};
+
+struct Context {
+    output: OutputBuffer,
+    flash: FlashSettings,
+}
+
+const ROOT_MENU: menu::Menu<Context> = menu::Menu {
+    label: "root",
+    items: &[
+        &menu::Item {
+            command: "reset",
+            help: Some("Reset Stabilizer to force new settings to take effect."),
```

Comment:
* Let's make this already device and application agnostic. `s/Stabilizer/device`.
* Let's also add a command that resets the settings to their default (and distinguish clearly which command resets the CPU and which resets the settings).
* Does the term `property` match the term we use in `miniconf`? I think there it's `item`.
* Let's also state clearly here that these settings here are distinct from the application `mqtt` settings. Let's open an issue to discuss how we can use/expose the application settings through usb and save/load them in flash to be overriden by mqtt.
* Also I'd like to consider making this menu interface to miniconf (plus maybe persistent storage) a agnostic thing (crate).